upnext/refresh.py

```diff
@@ -1,31 +1,27 @@
-import spotipy
 from django.conf import settings
 import time
 import datetime
 from threading import Thread
-import spotipy.util as util
 import spotipy.oauth2 as oauth2
 import tokens
 
 
 class Refresh(Thread):
-  def refresh_tokens(self):
-    now = time.strftime("%c")
-    print "I refreshed at", now
```

Comment:
can we keep the now = time... and print statement for debugging purposes on localhost + making sure it runs properly